src/style.css

```diff
@@ -1 +1,91 @@
-/* Styles go here */
+@font-face {
+  font-family: "Roboto";
+  src: url(./fonts/Roboto-Medium.ttf);
+  font-weight: 500px;
+}
+
+@font-face {
+  font-family: "Roboto-bold";
+  src: url(./fonts/Roboto-Bold.ttf);
+  font-weight: 800;
+}
+
+.header {
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  justify-content: space-between;
+
+  padding: 0 50px;
+  width: 100;
+  height: 60px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header:hover {
+  height: 60px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__logo {
+  padding: 10px 0;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+.nav__item {
+  margin-right: 20px;
+  padding: 23px 0;
+}
+
+.nav__item:last-of-type {
+  margin-right: 0;
+}
+
+.nav__link {
+  color: black;
+  text-decoration: none;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link:last-child {
+  margin: 0;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+  text-decoration: none;
+}
+
+.is-active::after {
+  position: absolute;
+  content: "";
+  width: 100%;
+  height: 4px;
+  left: 0;
+  right: 0;
+  top: 33px;
```

Comment:
use bottom: 0 instead of top: 33px

Reply:
I made a mistake, I didn't re-upload new links to my work...

Reply:
I have a completely different result in the browser than on the screens in the readme folders

Reply:
and if i try switch top on bottom my line hidde